src/globals/objects/_grid.scss

```diff
@@ -1,53 +1,78 @@
-@mixin govuk-grid {
-  @include govuk-clearfix;
-  margin-right: - ($govuk-gutter-half);
-  margin-left: - ($govuk-gutter-half);
-}
+// A mixin for a grid row
+// Creates a grid row with a standardised margin.
+// Usage:
+// Default:
+// @include @include govuk-grid()
+//
+// Provide a custom grid class:
+// @include govuk-grid("app-grid");
 
-@mixin govuk-grid-item {
-  box-sizing: border-box;
-  padding: 0 $govuk-gutter-half;
+@mixin govuk-grid($class: null) {
+  $class: "govuk-grid-row" !default;
 
-  @include mq($from: tablet) {
-    float: left;
+  .#{$class} {
+    @include govuk-clearfix;
+    margin-right: - ($govuk-gutter-half);
+    margin-left: - ($govuk-gutter-half);
   }
 }
 
-@include govuk-exports("grid") {
+// A mixin for a grid column
+// Creates a cross browser grid column with a standardised gutter between the
+// columns. Widths should be defined as fractions of the full desktop width
```

Comment:
Suggestion: "units" instead of "fractions"

Reply:
i like units